package.json

```diff
@@ -23,7 +23,7 @@
         "@aws-sdk/client-secrets-manager": "^3.195.0",
         "@aws-sdk/client-sqs": "^3.137.0",
         "@balancer-labs/sdk": "github:beethovenxfi/balancer-sdk#beethovenx-master",
-        "@balancer/sdk": "^0.12.0",
```

Comment:
did you check whether this breaks anything else? Not quite sure where the old sdk is used.

Reply:
I did check, the core of the SDK is used only in the SOR, there are some of it being used on the fetch-tokenpair-data and pool-on-chain-tokenpair-data, but it's only constants(WAD and ZERO_ADDRESS) and MathSol, that didn't suffer any changes in these latest updates